src/guide/frontend/android.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Android Client SDK
+---
+
+# Android Client SDK
+
+The Passwordless.dev Android client SDK gives users the ability to leverage their device’s built-in fingerprint sensor and/or FIDO security keys for secure passwordless access to websites and native applications that support the FIDO2 protocols
+
+## Requirements
+
+- Android 9.0 (API level 28) or higher
+- Java 8 or higher
+- [Having completed the 'Get started' guide](../get-started.md).
+
+## Installation
+
+::: tabs
+
+@tab Apache Maven
+
+```xml
+<dependency>
+  <groupId>com.bitwarden</groupId>
+  <artifactId>passwordless-android</artifactId>
+  <version>1.0.1</version>
+</dependency>
+```
+
+@tab Gradle Kotlin DSL
+
+```kotlin
+implementation("com.bitwarden:passwordless-android:1.0.1")
+```
+
+@tab Gradle Groovy DSL
+
+```groovy
+implementation 'com.bitwarden:passwordless-android:1.0.1'
+```
+
+:::
+
+## Permissions
+
+In your `AndroidManifest.xml`, you will need to add the following permissions:
+
+```xml
+<uses-permission android:name="android.permission.INTERNET" />
+```
+
+## Configuration (Android application)
+
+```kotlin
+data class PasswordlessOptions(
+   // Your public API key
+   val apiKey: String,
+
+   // Identifier for your server, for example 'example.com' if your backend is hosted at https://example.com.
+   val rpId: String,
+
+   // This is where your Facet ID goes
+   val origin: String,
+
+   // Where your backend is hosted
+   val backendUrl:String,
+
+   // Passwordless.dev server, change for self-hosting
+   val apiUrl: String = "https://v4.passwordless.dev"
+)
+```
+
+### .well-known/assetlinks.json
+
+In your application's `AndroidManifest.xml`, you will then need to add the tag below under `manifest::application`:
+
+```xml
+<meta-data
+  android:name="asset_statements"
+  android:resource="@xml/assetlinks" />
+```
+
+In your application's `res/xml/assetlinks.xml`, you will then need to add the following content. This will tell our Android application where our `.well-known/assetlinks.json` file is hosted.
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="assetlinks">https://yourexample.com/.well-known/assetlinks.json</string>
+</resources>
+```
+
+### Facet ID
+
+The `Facet ID` will be used at a later point in this guide to use as the `origin`.
```

Comment:
This isnt an error necessarily, but sometimes shortening things can help them be more understandable. For example changing the line here to "will be used later in this guide as the `origin`. "